roles/github/defaults/main.yml

```diff
@@ -73,53 +78,79 @@ github_run_kolla_config_diff:
 github_run_infra_vm_host_configure:
   file_name: run-infra-vm-host-configure.yml
   trigger:
-    workflow_dispatch: *github_kayobe_inputs
+    workflow_dispatch:
+      - "{{ github_kayobe_limit_input }}"
+      - "{{ github_kayobe_tags_input }}"
   arguments:
+    - KAYOBE_AUTOMATION_SSH_PRIVATE_KEY
```

Comment:
There's a lot of duplication of these arguments. Refactoring into a couple of variables would help to avoid this issue of missing arguments.

Reply:
This new approach should improve the situation.